src/utils.ts

```diff
@@ -381,6 +381,8 @@ export const supportsEmoji = () => {
   ctx.fillStyle = "#f00";
   ctx.textBaseline = "top";
   ctx.font = "32px Arial";
-  ctx.fillText("\ud83d\udc28", 0, 0); // U+1F428 KOALA
+  // Modernizr used 🐨, but it is sort of supported on Windows 7.
+  // Luckily 😀 isn't supported.
+  ctx.fillText("😀", 0, 0);
```

Comment:
Looks supported for me

![image](https://user-images.githubusercontent.com/5153846/107067012-fa9e2180-67de-11eb-839f-9757bdc4d62b.png)

Reply:
but it seems to work in-app. How does the detection work exactly?

Reply:
it seems like that even though `😀` is displayed in some form on Win7, the `ctx.getImageData().data` returns `[0,0,0,0]`. Strange.

For what it's worth, shouldn't we test the emoji(s) we care about?

Reply:
Tested on BrowserStack that the feature test fails on Windows 7 and works elsewhere.